pkg/Microsoft.ML.OnnxTransform/Microsoft.ML.OnnxTransform.nupkgproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>ML.NET component for Sonoma</PackageDescription>
```

Comment:
>Sonoma [](start = 45, length = 6)

Is Sonoma a public name?

Reply:
replaced with microsoft.ml.scoring

---
In reply to: [220280263](https://github.com/dotnet/machinelearning/pull/942#discussion_r220280263) [](ancestors = 220280263)